lib/l10nUtil.js

```diff
@@ -65,14 +72,17 @@ module.exports.rebaseBraveStringFilesOnChromiumL10nFiles = (path) =>
     [chromiumSettingsStringsPartPath]: braveSettingsStringsPartPath,
     [chromiumMdExtensionsPartPath]: braveMdExtensionsPartPath,
     [chromiumExtensionsResourcesPath]: braveExtensionsResourcesPath,
-    [chromiumSettingsResourcesPath]: braveSettingsResourcesPath
+    [chromiumSettingsResourcesPath]: braveSettingsResourcesPath,
+    [chromiumBrowserResourcesPath]: braveBrowserResourcesPath
   }).forEach(([sourcePath, destPath]) =>
     fs.writeFileSync(destPath,
       fs.readFileSync(sourcePath, 'utf8')
         .replace(/<\?xml version="1.0" encoding="utf-8"\?>/i, '<?xml version="1.0" encoding="utf-8"?>\n' + autoGeneratedWarning)
         .replace('<structure name="IDR_MD_EXTENSIONS_SIDEBAR_HTML"', '<structure name="IDR_MD_EXTENSIONS_SIDEBAR_HTML" preprocess="true"')
         .replace('<structure name="IDR_SETTINGS_APPEARANCE_FONTS_PAGE_HTML"', '<structure name="IDR_SETTINGS_APPEARANCE_FONTS_PAGE_HTML" preprocess="true"')
         .replace('<structure name="IDR_SETTINGS_PASSWORDS_SECTION_HTML"', '<structure name="IDR_SETTINGS_PASSWORDS_SECTION_HTML" preprocess="true"')
+        .replace('<include name="IDR_MD_HISTORY_SIDE_BAR_HTML"', '<include name="IDR_MD_HISTORY_SIDE_BAR_HTML" flattenhtml="true"')
+        .replace(pageVisibility, bravePageVisibility + pageVisibility)
```

Comment:
It occurred to me that we should probably throw an error if the expected number of replacements doesn't match exactly with the number of replacements. In this case 1.
I posted this `Backlog` issue for that:
https://github.com/brave/brave-browser/issues/475